app/models/task.py

```diff
@@ -1,5 +1,20 @@
 from app import db
+from flask import jsonify
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True, default=None)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'))
```

Comment:
A very small detail, but you've been using double quotes throughout your model classes, so make sure you stick with that.

```suggestion
    goal_id = db.Column(db.Integer, db.ForeignKey("goal.goal_id"))
```